docs/shop_administration/catalogs.md

```diff
@@ -23,14 +23,15 @@ To create a catalog:
 
 ### Catalog filters
 
-You can add the following filters to select products for a catalog from the **Add filter** menu:
-
-- Product Category
-- Product Created (date when the product was created)
-- Product availability
-- Product code
-- Product price (in selected currency, with the option to set minimum and maximum price)
-- Product type
+When you create a new catalog, all products are included in it by default.
+You can narrow down this selection by applying the following filters from the **Add filter** menu:
+
+- Category
+- Created (date when the product was created)
+- Availability
+- Code
+- Price (in selected currency, with the option to set minimum and maximum price)
+- Type
 - and additionally, filters based on all configured product attributes
 
```

Comment:
the meta "description" has some bug probably as it is shown in the generated output in read the docs

Reply:
true, meta must be written in one line.